### PR TITLE
Enable automatic deploy to DNAnexus on GitHub release

### PR DESCRIPTION
**Pull request by adthrasher**
Automatically update DX version of CICERO. Build and push the app. Also point app to GitHub packages Docker image rather than packaging in the DX app.

### CODE REVIEW

**Comment by adthrasher** [comment]
@jordan-rash - Could you take a look at this as a sanity check?

File: dx_app/src/code.sh [hunk 1]
```diff
@@ -105,14 +105,6 @@ main() {
...
-
-  echo "  [*] Loading container image ..."
-  image_tarfile_path=/stjude/docker/cicero-docker.tar
-  if [ -e $image_tarfile_path.gz ]
-  then gunzip $image_tarfile_path.gz
-  fi
-  docker load -i $image_tarfile_path
-   
   echo "  [*] Determine image ID ..."
   image_id=`docker images -q stjude/cicero | head -n 1`
```
Comment (jordan-rash) [inline dx_app/src/code.sh:109]: Should this be pointed at docker hub?

**Review by jordan-rash — COMMENTED** [review]
Mostly LGTM.  Just the one comment on the docker hub reference

File: dx_app/src/code.sh (hunk 1 above)
Comment (adthrasher) [reply dx_app/src/code.sh]: Thanks, good catch. That was legacy code as we used to package the Docker image in the app and then loaded from the tarball. Now The run command points to the tagged version in GitHub packages.

**Pull request merged**